blast/static/blast/scripts/organism-admin.js

```diff
@@ -24,21 +24,15 @@
             $('#id_short_name').data('userModified', false);
         }
     });
-    // Get NCBI taxonomy id
+
     $('#id_display_name').change(function () {
         var name = $('#id_display_name').val().toLowerCase().replace(/^\s+|\s+$/g, '');
-        $.ajax({
-            url: 'http://eutils.ncbi.nlm.nih.gov/entrez/eutils/esearch.fcgi',
-            data: {
-                db: 'taxonomy',
-                term: name,
-                retmode: 'json',
-            }
-        }).done(function (data) {
-            //console.log(data['esearchresult']['idlist']);
-            $('#id_tax_id').val(data['esearchresult']['idlist'][0])
+        $.getJSON('https://eutils.ncbi.nlm.nih.gov/entrez/eutils/esearch.fcgi?db=taxonomy&retmode=json&term=' + name, function (data) {
```

Comment:
So ..... the original problem happens because we need to request the search through `https` rather than `http`, am I right?